modules/claims_api/lib/bgs_service/corporate_update_web_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class CorporateUpdateWebService < ClaimsApi::LocalBGS
+    def bean_name
+      'CorporateUpdateServiceBean'
+    end
+
+    def corporate_update
+      self
+    end
```

Comment:
This method is not strictly required but in order to keep the call exactly the same I added it, if we feel like that is not important I think we certainly could remove it and just call the method differently in the job and tests.